src/common/snippets/tests/src/pass/gn_decomposition.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+#include "pass/gn_decomposition.hpp"
+#include "common_test_utils/common_utils.hpp"
+#include "subgraph_group_normalization.hpp"
+#include "subgraph_lowered.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+std::string GNDecompositionTest::getTestCaseName(testing::TestParamInfo<GroupNormalizationParams> obj) {
+    PartialShape input_shape;
+    size_t num_group;
+    float eps;
+    std::tie(input_shape, num_group, eps) = obj.param;
+    std::ostringstream result;
+    result << "IS=" << ov::test::utils::partialShape2str({input_shape}) << "_";
+    result << "num_group=" << num_group << "_";
+    result << "eps=" << eps;
+    return result.str();
+}
+
+void GNDecompositionTest::SetUp() {
+    LoweringTests::SetUp();
+    std::vector<PartialShape> input_shapes;
+    PartialShape data_shape;
+    size_t num_group;
+    float eps;
+    std::tie(data_shape, num_group, eps) = this->GetParam();
+    PartialShape scaleShiftShape = PartialShape{data_shape[1]};
```

Comment:
I think we should check for `data_shape_rank >= 2` here. What do you think?

Reply:
added, thanks!